src/models/smtp.js

```diff
@@ -37,6 +34,23 @@ export default function(sequelize, DataTypes) {
                 });
 
             }
+        },
+        classMethods: {
+            smtpTest(email) {
+                return new Promise((resolve, reject) => {
+                    this.findOne({ where: { email: email } })
+                        .then((data) => {
+                            if (data) {
+                                sequelize.query(`UPDATE SMTP  SET status = (case when email = "${email}" then 1 else 0 end);`)
```

Comment:
then its very simple, simply fire update and all status = 0 for all rows

and this update status = 1 where email = incoming email